### PR TITLE
SC2: 0.4.3 bugfixes

### PR DESCRIPTION
**Pull request by Ziktofel**
## What is this fixing or adding?
Fixes few minor generation-related bugs:

## How was this tested?
Generated few affected games and some other ones, launched the client and connected to the slot
See also: 
https://github.com/Ziktofel/Archipelago/pull/59
https://github.com/Ziktofel/Archipelago/issues/58
https://github.com/Ziktofel/Archipelago/pull/66

### CODE REVIEW

File: worlds/sc2wol/Options.py [hunk 1]
```diff
@@ -42,7 +42,8 @@ class FinalMap(Choice):
     Vanilla mission order always ends with All in mission!
 
     Warning: Using All-in with a short mission order (7 or fewer missions) is not recommended,
-        as there might not be enough locations to place all the required items.
+        as there might not be enough locations to place all the required items,
+        any excess required items will be placed into starting inventory!
```
Comment (MatthewMarinets) [inline worlds/sc2wol/Options.py:46]: Missing an article, maybe "items will be placed into [the player's] starting inventory"

**Review by MatthewMarinets — APPROVED** [review]
Looks good to me.

**Comment by ThePhar** [comment]
Since you changed the name of some of the locations, please increment `data_version` by one. Hopefully only a little while longer until we can completely drop `data_version` support from core.

**Comment by Ziktofel** [comment]
Done

**Review by ThePhar — APPROVED** [review]

**Pull request merged**